cms/static/cms/js/modules/cms.plugins.js

```diff
@@ -134,6 +137,22 @@
                     );
                 });
 
+                // prevents single click from messing up the edit call
```

Comment:
hmm, but now it's basically impossible to edit link or button. We have to decide upon the behavior then